lib/src/utils/utils.dart

```diff
@@ -1,31 +1,57 @@
+import 'package:flutter/foundation.dart';
 import 'package:url_launcher/url_launcher.dart';
 
 import '../models/wallet.dart';
 
 class Utils {
+  static bool isIOS() => defaultTargetPlatform == TargetPlatform.iOS;
```

Comment:
@elliotsayes Can you please turn these into properties so it's in line with `TargetPlatform` properties?